crates/gourgeist/src/bare.rs

```diff
@@ -174,9 +156,19 @@ pub fn create_bare_venv(
     }
 
     // Add all the activate scripts for different shells
-    // TODO(konstin): RELATIVE_SITE_PACKAGES is currently only the unix path. We should ensure that all launchers work
-    // cross-platform.
     for (name, template) in ACTIVATE_TEMPLATES {
+        let relative_site_packages = if cfg!(unix) {
+            format!(
+                "../lib/{}{}.{}/site-packages",
+                interpreter.site_packages_python(),
+                interpreter.python_major(),
+                interpreter.python_minor(),
+            )
+        } else if cfg!(windows) {
+            "../Lib/site-packages".to_string()
```

Comment:
I assume this is right? (This is the only behavior change in the PR -- I tried to do the above TODO.)